### PR TITLE
Added OpenUxAS to platforms

### PR DESCRIPTION
**Pull request by Ikutania**

### CODE REVIEW

**Comment by Ikutania** [comment]
Added the Dockerfile for OpenUxAS

**Comment by Ikutania** [comment]
Added the Dockerfile for UxAS

File: platforms/OpenUxAS/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RUN apt-get update && \
+    apt-get install -y keyboard-configuration
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update												\
+    && apt-get install -y											\
+           build-essential											\
+           cmake												\
+           git													\
```
Comment (obicons) [inline platforms/OpenUxAS/Dockerfile:12]: nit: make sure the backslashes are vertically aligned in Dockerfiles

File: platforms/OpenUxAS/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+           lsb-release												\
+           wget													\
+	   software-properties-common										\
+           sudo													\
+    && wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -                                \
+    && add-apt-repository "deb http://apt.llvm.org/focal/     llvm-toolchain-focal$LLVM_VERSION_STRING    main" \
+    && apt-get update                                                                                           \
+    #&& apt-get install -y libclang-13-dev									\
```
Comment (obicons) [inline platforms/OpenUxAS/Dockerfile:20]: I need to remove this commented-out line from PX4. Go ahead and remove it from this file.

File: platforms/OpenUxAS/Dockerfile (hunk 1 above)
Comment (Ikutania) [reply platforms/OpenUxAS/Dockerfile:12]: Not certain why they aren't showing up vertically aligned on the comment. When I open the file to edit, they are aligned.

File: platforms/OpenUxAS/Dockerfile (hunk 2 above)
Comment (Ikutania) [reply platforms/OpenUxAS/Dockerfile]: Removed

File: platforms/OpenUxAS/Dockerfile (hunk 1 above)
Comment (obicons) [reply platforms/OpenUxAS/Dockerfile:12]: There might be a tabs v. space issue. You may try deleting all the whitespace and realigning.

**Review by obicons — COMMENTED** [review]
Squash the two commits into a single commit to keep the history clean. See https://www.git-tower.com/learn/git/faq/git-squash for help.

**Pull request closed without merging**